templates/_layout.html

```diff
@@ -14,8 +14,36 @@
       <meta property="og:site_name" content="Snapcraft"/>
       <meta property="og:type" content="website"/>
       <meta property="og:description" content="{{ self.description() }}"/>
-      <meta property="og:image" content="https://assets.ubuntu.com/v1/e635d1ef-snapcraft_green-red_hex.png" />
+      <meta property="og:image" content="{% block meta_image %}https://assets.ubuntu.com/v1/e635d1ef-snapcraft_green-red_hex.png{% endblock %}" />
+      <meta property="og:url" content="https://snapcraft.io/{% block snap %}{% endblock %}" />
+      <meta property="twitter:card" content="summary_large_image" />
       <meta property="twitter:site" content="@snapcraftio" />
+      <meta property="twitter:creator" content="@snapcraftio" />
+      <meta property="twitter:title" content="{{ self.title() }}" />
```

Comment:
At least for some of them (title, description, image, etc) we shouldn't need to duplicate values for twitter as they seem to default to values form `og:`: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup

Reply:
amaze :+1: